lib/ecto/repo/schema.ex

```diff
@@ -18,6 +18,10 @@ defmodule Ecto.Repo.Schema do
     do_insert_all(repo, adapter, nil, {nil, table}, rows, opts)
   end
 
+  def insert_all(repo, adapter, table, rows, opts) when is_tuple(table) do
```

Comment:
Can we check for `{_prefix, _source} = table` instead of the `is_tuple` guard? Thank you! :heart: